### PR TITLE
Add cluster into Proto struct

### PR DESCRIPTION
**Pull request by alebaffa**

### CODE REVIEW

File: pallets/protos/src/lib.rs [hunk 1]
```diff
@@ -426,7 +429,7 @@ pub mod pallet {
 			tags: BoundedVec<BoundedVec<u8, <T as pallet::Config>::StringLimit>, T::MaxTags>,
 			linked_asset: Option<LinkedAsset>,
 			license: UsageLicense<T::AccountId>,
-			_cluster: Option<Cluster<T::AccountId>>,
+			cluster: Option<Cluster<T::AccountId>>,
```
Comment (sinkingsugar) [inline pallets/protos/src/lib.rs]: Err, isn't that storing the whole big structure?

**Review by sinkingsugar — CHANGES_REQUESTED** [review]

File: pallets/protos/src/lib.rs (hunk 1 above)
Comment (alebaffa) [reply pallets/protos/src/lib.rs]: Changed it to store only the cluster_id, i.e. `Hash128`.

Comment (sinkingsugar) [reply pallets/protos/src/lib.rs]: Btw, otherwise made no sense, it would copy the data and leave it into a detached state. You didn't consider that?

**Review by sinkingsugar — APPROVED** [review]

**Review by sinkingsugar — APPROVED** [review]
Now it's how it should have been 👍
The previous version was concerning though, I still don't get what was the point on storing the full struct copy in protos 🤔

**Comment by alebaffa** [comment]
It was a quick code I made locally that was not intended to be pushed, I know it made no sense.

**Comment by sinkingsugar** [comment]
@alebaffa **devel** was with `_cluster: Option<Cluster<T::AccountId>>,` for a while, I didn't notice though 😄 let's pay more attention or when we do an audit it will be a tragedy. I can't be always alert with reviews.

**Pull request merged**